tools/obs/main.go

```diff
@@ -34,60 +34,39 @@ func main() {
 	fmt.Println(data)
 }
 
-func getShardV1Data() *outport.ArgsSaveBlockData {
-	return &outport.ArgsSaveBlockData{
-		HeaderHash: []byte("hash2"),
-		Body: &block.Body{
-			MiniBlocks: []*block.MiniBlock{
-				{
-					TxHashes:        [][]byte{},
-					ReceiverShardID: 1,
-					SenderShardID:   1,
-					Type:            0,
-				},
+func getShardV1Data() *notifierData.SaveBlockData {
+	return &notifierData.SaveBlockData{
+		Hash: "blockHash",
+		Txs: map[string]*transaction.Transaction{
+			"hash1": {
+				Nonce: 1,
 			},
 		},
-		Header: &block.Header{
-			ShardID:   1,
-			TimeStamp: 1234,
```

Comment:
Why where the other fields deleted here?

Reply:
it was an old structure which included the Header, but it's not available from core anymore, changed to use the internal structure that's being used on notifier for unmarshalling; last time when we did changes to api we've handled both versions based on payload data, that's why there is V1 and V2, not directly related to shard header type, renamed functions to be more clear